### ISSUE TITLE
IMOS: check_attributes - obscure error message 

### ISSUE THREAD

**Issue by lbesnard** [aodn/compliance-checker#40]
checker output

```
    quality_variable                   :3:     3/ 6 :  
        LATITUDE_quality_control       :3:     0/ 1 :  
            check_attributes           :3:     0/ 1 : Variable attribute is not
                                                      present                                                         
        LONGITUDE_quality_control      :3:     0/ 1 :                                                                 
            check_attributes           :3:     0/ 1 : Variable attribute is not                                       
                                                      present
        PSAL_quality_control           :3:     3/ 4 :                                                                 
            check_attributes           :3:     0/ 1 : Variable attribute is not                                       
                                                      present
```

Message doesn't contain anything meaningful to me. the string 'is not present' can't be found in the imos checker cause it comes from an external lib. So it makes difficult to know what's the error message is about. 

The file doesn't follow obviously the IMOS convention, but this test doesn't help to know what to fix imo
tested file :
http://thredds.aodn.org.au/thredds/catalog/IMOS/SOOP/SOOP-TRV/VMQ9273_Solander/By_Cruise/Cruise_START-20150105T073551Z_END-20150106T024945Z/salinity/catalog.html?dataset=IMOS/SOOP/SOOP-TRV/VMQ9273_Solander/By_Cruise/Cruise_START-20150105T073551Z_END-20150106T024945Z/salinity/IMOS_SOOP-TRV_S_20150105T073551Z_VMQ9273_FV01_END-20150106T024945Z_C-20150106T190156Z.nc

**Comment by fxmzb123** [comment]
@lbesnard fixed and see pull request #47

**Comment by lbesnard** [comment]
@fxmzb123 thanks it's working fine
